vcd/resource_vcd_vapp_org_network.go

```diff
@@ -0,0 +1,301 @@
+package vcd
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	"log"
+	"strings"
+)
+
+func resourceVcdVappOrgNetwork() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceVappOrgNetworkCreate,
+		Read:   resourceVappOrgNetworkRead,
+		Update: resourceVappOrgNetworkUpdate,
+		Delete: resourceVappOrgNetworkDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceVcdVappOrgNetworkImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"vapp_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"org": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Description: "The name of organization to use, optional if defined at provider " +
+					"level. Useful when connected as sysadmin working across different organizations",
+			},
+			"vdc": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The name of VDC to use, optional if defined at provider level",
+			},
+			"org_network": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
- Missing Description.
- From the field name it's not clear whether we are referencing the name or ID. If name, then I suggest naming this field `org_network_name` like we have `network_name` in NAT rule.

CC: @Didainius @dataclouder

Reply:
added description

Reply:
Thanks! Please add "is":
> Organization network name to which vApp network **is** connected to

Reply:
Thank you. fixed